cypress/e2e/39-projects.cy.ts

```diff
@@ -220,7 +220,8 @@ describe('Projects', { disableAutoLogin: true }, () => {
 				.should('contain.text', 'Notion account personal project');
 		});
 
-		it('should move resources between projects', () => {
+		// eslint-disable-next-line n8n-local-rules/no-skipped-tests
```

Comment:
This kind of beats the point of the eslint rule. If there is a ticket to fix the test, why not just delete the test?

Reply:
I thought we wanted to skip it to unblock people quickly and then look at fixing it ASAP.

Reply:
@cstuncsik is going to split up the test instead.

Reply:
yes, let's see if splitting helps
as we talked about earlier it uses too many resources and hopefully, multiple test cases help with that